### PR TITLE
feat: add `tccutil` spec

### PR DESCRIPTION
**Pull request by SeparateRecords**
Not that this matters much since if you need to use `tccutil` Fig probably isn't working, but it's a fun and small spec!

image

Getting a list of bundle IDs is a surprisingly difficult task... I'll reuse this generator for `open -b`
image

### CODE REVIEW

File: src/tccutil.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  {
+    name: "DeveloperTool",
+    description:
+      "Allows app to run software locally that do not meet the system’s security polic",
+  },
+].map((item) => Object.assign(item, { icon: "⚙️" }));
+
+const generateBundleIds = (path: string): Fig.Generator => ({
+  scriptTimeout: 15000,
+  cache: { ttl: 1000 * 60 * 60 * 24 },
```
Comment (fedeci) [inline src/tccutil.ts:143]: We can use swr here. It will guarantee us to always get some data (stale or not).

Comment (SeparateRecords) [reply src/tccutil.ts]: Done!

Comment (fedeci) [reply src/tccutil.ts]: We can also keep the ttl to avoid rerunning the script in background each time :)

**Review by mschrage — APPROVED** [review]

**Comment by mschrage** [comment]
@SeparateRecords I'll merge this in but can you fix lint error @SeparateRecords (I know it wasn't from your PR)

**Pull request merged**